.gitlab/junit_upload/junit_upload.yml

```diff
@@ -0,0 +1,70 @@
+---
+unit_tests_arm64_windows_junit_upload:
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
We cannot install the cli on the image used on windows, can we?

Reply:
We can but I tried and add many issues so I wanted to de-correlate. I created [ACIX-192](https://datadoghq.atlassian.net/browse/ACIX-192) for this